docs/contributors/repository-management.md

```diff
@@ -70,6 +70,14 @@ Gutenberg follows a feature branch pull request workflow for all code and docume
 2. Make your changes, testing thoroughly.
 3. Commit your changes when you’re happy with them, and push the branch.
 4. Open your pull request.
+5. Label and name your pull request appropriately (see below).
+
+For labeling and naming pull requests, here are guidelines to follow that make compiling the changelog more efficient and organized:
```

Comment:
how can we both have these guidelines or maybe "recommendation" and also avoid having to force contributors to learn dozens of things in order to try opening a PR.

Should we clarify that it applies essentially for regular contributors?
Should we make it clear that it's totally fine to make mistakes and learn as you do?

Reply:
Just updated this section to make the language more of a recommendation specifically for regular contributors and to be more "mistake friendly" :) Resolving this but let me know if the new language still doesn't sit right.